src/docs/arbitrum-one.mdx

```diff
@@ -117,6 +117,7 @@ links:
     | 42 | TIMESTAMP | `block.timestamp` | Timestamp of the L2 block | Timestamp of the L1 block <Modified /> |
     | 43 | NUMBER | `block.number` | Returns an "estimate" of the L1 block number at which the Sequencer received the transaction (see [Block Numbers and Time](https://developer.arbitrum.io/time)) | Returns the L1 block number <Modified /> |
     | 44 | DIFFICULTY | `block.difficulty` and `block.prevrandao` | Returns the constant `2500000000000000` | Returns the output of the randomness beacon provided by the beacon chain <Modified /> |
+    | 4a | BLOBBASEFEE | `block.blobbasefee` | Unsupported. If the opcode is encountered, the transaction will be reverted. | Returns current block’s blob base fee.  <Unsupported /> |
```

Comment:
What about `BLOBHASH` and the new [precompile](https://www.evm.codes/precompiled#0x0a?fork=cancun)?

Reply:
Only `BLOBBASEFEE` is not supported ([github link](https://github.com/OffchainLabs/go-ethereum/blob/master/core/vm/eips.go#L288)). The code for `BLOBHASH` and the pointEvaluation precompile is unmodified from the geth repo

Reply:
What would happen if the `blobhash` is called? I am pretty sure that it won't work. 
Can you try both of them in remix so that we are sure